hack/build.sh

```diff
@@ -16,6 +16,8 @@
 
 set -o pipefail
 
+source $(dirname $0)/../scripts/test-infra/library.sh
```

Comment:
Unfortunately, that won't work for macOS. See https://github.com/knative/test-infra/issues/2143

`build.sh` is supposed to run on any Unix like OS, also macOS, _without_ any other dependencies. So having Bash 4 dependency here is a no-go, so we can include any test-infra scripts in build.sh.